packages/language-server/test/snapshots/xml-view-diagnostics/snapshots-utils.ts

```diff
@@ -93,7 +93,8 @@ export function readSnapshotDiagnosticsLSPResponse(
 }
 
 const ui5ModelPromise = generateModel({
-  version: "1.71.14",
+  framework: "sapui5",
```

Comment:
longer term we may want to extract this "MAGIC" string of the version if it reused in all/most tests.

Reply:
Sure, we would in general need to think about a more dynamic handling of the versions.